npm/npm_import.bzl

```diff
@@ -79,6 +79,33 @@ def npm_translate_lock(
     Thus it is performant to convert a very large pnpm-lock.yaml file without concern for
     users needing to fetch many unnecessary packages.
 
+    **Compatabilty with pnpm***
+
+    The node_modules layed out by rules_js should be bug-for-bug compatible with the node_modules that
+    pnpm lays out with [hoisting](https://pnpm.io/npmrc#hoist) disabled (`hoist=false` set in your `.npmrc`).
+
+    If you hit import/require runtime failures in 3rd party npm packages when using rules_js that you do not get
+    outside of Bazel when using pnpm, we recommend adding `hoist=false` to your `.npmrc`:
+
+    ```
+    echo "hoist=false" >> .npmrc
+    ```
+
+    This will prevent pnpm from creating the hidden `node_modules/.pnpm/node_modules` folder with hoisted
+    dependencies which allow for packages to depend on "phantom" undeclared dependencies. In most cases, the failure
+    you see in rules_js will be reproducable outside of Bazel when hoisting is disabled.
```

Comment:
nit: check spelling "reproducible"

Reply:
hmmm. my spellcheck plugin in VSCode seems to be malfunctioning